fmriprep/workflows/tests/test_base.py

```diff
@@ -105,7 +105,7 @@ def bids_root(tmp_path_factory):
 
 
 def _make_params(
-    bold2t1w_init: str = "register",
+    bold2anat_init: str = "t1w",
```

Comment:
The default is "auto", so that's what we should do here:

```suggestion
    bold2anat_init: str = "auto",
```